dev/devicelab/bin/tasks/build_android_host_app_with_module_aar.dart

```diff
@@ -308,17 +305,6 @@ class ModuleTest {
         return TaskResult.failure("Debug host APK doesn't contain metadata: flutterProjectType = module ");
       }
 
-      final String analyticsOutput = analyticsOutputFile.readAsStringSync();
-      if (!analyticsOutput.contains('cd24: android')
-          || !analyticsOutput.contains('cd25: true')
-          || !analyticsOutput.contains('viewName: assemble')) {
-        return TaskResult.failure(
-          'Building outer app produced the following analytics: "$analyticsOutput" '
-          'but not the expected strings: "cd24: android", "cd25: true" and '
-          '"viewName: assemble"'
-        );
-      }
```

Comment:
This is already under test in https://github.com/flutter/flutter/blob/125b4e945e7e9d8bbbd9276b1a2d9162bd09696a/packages/flutter_tools/test/commands.shard/permeable/build_bundle_test.dart#L69